<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletWindow</class>
 <widget class="QWidget" name="WalletWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="laName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="laDate">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>51</y>
     <width>190</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="laValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bDelete">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bIncome">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Transaction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bSettings">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QTableView" name="transactions">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>181</y>
     <width>691</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
